src/pages/docs/api-reference/[slug].tsx

```diff
@@ -19,31 +19,25 @@ const APIPage: NextPage<Props> = ({ slug }) => {
         strategy="beforeInteractive"
       />
 
-      <rapi-doc
-        spec-url={`/docs/api-reference/${slug}.json`}
-        style={rapidocstyle}
-        fill-request-fields-with-example={true}
-        theme="light"
-        render-style="focused"
-        bg-color="#FFFFFF"
-        nav-bg-color="#FFFFFF"
-        nav-hover-bg-color="#F8F7FC"
-        nav-hover-text-color="#000711"
-        show-method-in-nav-bar={true}
-        primary-color="#142032"
-        regular-font="VTEX Trust Variable"
-        mono-font="VTEX Trust Variable"
-        load-fonts={false}
-        use-path-in-nav-bar={false}
-        nav-text-color="#4A596B"
-        nav-accent-color="#D71D55"
-        nav-item-spacing="relaxed"
-        on-nav-tag-click="expand-collapse"
-        schema-style="table"
-        schema-description-expanded={true}
-        show-info={true}
-        show-header={false}
-      ></rapi-doc>
+      <Flex sx={styles.container}>
+        <Sidebar sectionSelected="API Reference" />
```

Comment:
I noticed the collapse sidebar button (the round one on the right of the image below) is not working in the API Reference pages. I think it is because you're missing the SidebarContextProvider here.

<img width="232" alt="image" src="https://user-images.githubusercontent.com/37647055/182447013-787ac643-4e38-471a-b44d-afee940e2533.png">

Reply:
👀

Reply:
👀

Reply:
Done!